src/components/global/footer.tsx

```diff
@@ -1,9 +1,263 @@
+  padding: 0;
+  border: 2px solid ${colors.blackLight};
+  transition: 0.3s ease;
+  border-radius: 0.75rem;
+  vertical-align: middle;
+
+  &:hover {
+    background-color: ${colors.accent};
+    border-color: ${colors.accent};
+  }
+`;
+
+const StyledLinkWrapper = styled.div`
+  display: flex;
+  height: 9px;
+  color: #F8F8F8;
+  font-weight: 400;
+  opacity: 0.5;
+  margin-bottom: 17px;
+  justify-content: end;
+
+  & > a {
+    margin-right: 1.5rem;
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    justify-content: center;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    justify-content: center;
+    font-size: 0.8rem;
+  }
+`;
+
+const StyledSlider = styled.div`
+  height: 155px;
+  margin: auto;
+  width: 100%;
+  display: grid;
+  place-items: center;
+  overflow: hidden;
+  padding-top: 38px;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    height: 103px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    height: 74px;
+  }
+`;
+
+const StyledSlideTrack = styled.div`
+  display: flex;
+  width: calc(1725px * 4); // total width of all slides
+  animation: scroll 8s linear infinite;
+
+  @keyframes scroll {
+    0% {
+      transform: translateX(0);
+    }
+    100% {
+      transform: translateX(-1725px);
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: calc(1000px * 4);
+    animation: scroll 6s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-1000px);
+      }
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: calc(600px * 4);
+    animation: scroll 4s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-600px);
+      }
+    }
+  }
+`;
+
+const StyledSlide = styled.div`
+  width: 1725px;
+  display: flex;
+  align-items: center;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: 1000px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: 600px;
+  }
+`;
+
+const StyledSlideText = styled.span`
+  font-weight: 600;
+  font-size: 344px;
+  letter-spacing: 0.6rem;
+  width: 100%;
+  line-height: 72px;
+  text-shadow: 9px 0px ${colors.accent};
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    font-size: 185px;
+    line-height: 40px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    font-size: 103px;
+    line-height: 21.6px;
+    text-shadow: none;
+  }
+`;
 
 export default function Footer() {
+  const isDesktop = useIsDesktop();
   return (
     <footer>
-      <h2>Footer will go here</h2>
+      <StyledWrapper>
+      <StyledContent>
+        <StyledParagraphWrapper>
+          <StyledTextSpacer>{"Interested?"}</StyledTextSpacer>
+          <StyledPillWrapper>
+            <Pill pillText={"Interested?"} />
+          </StyledPillWrapper>
+          <StyledParagraphText>
```

Comment:
One problem here - nice that you use an existing component to reduce code written, but I think on mobile this text is too small - it would need these styles overwritten on mobile

```
font-size: ${getRemSize(dimensions.headingSizes.display2.mobile)};
line-height: 72px;
```